pkg/agent/cniserver/server_test.go

```diff
@@ -330,16 +331,21 @@ func TestValidateOVSPort(t *testing.T) {
 	containerConfig.OVSPortConfig = &agent.OVSPortConfig{PortUUID: portUUID, IfaceName: hostIfaceName}
 
 	ifaceStore.AddInterface(hostIfaceName, containerConfig)
-	err := validateOVSPort(ifaceStore, hostIfaceName, containerMACStr, containerID, result.IPs)
+	err := podConfigurator.validateOVSPort(hostIfaceName, containerMACStr, containerID, result.IPs)
 	assert.Nil(t, err, "Failed to validate OVS port configuration")
 }
 
 func TestRemoveInterface(t *testing.T) {
 	controller := gomock.NewController(t)
 	defer controller.Finish()
-	ifaceStore := agent.NewInterfaceStore()
 	mockOVSBridgeClient := ovsconfigtest.NewMockOVSBridgeClient(controller)
 	mockOFClient := openflowtest.NewMockClient(controller)
+	ifaceStore := agent.NewInterfaceStore(nil)
```

Comment:
would there be problem if using nil as the func? I see the code doesn't check if containerInterfaceParser is nil when using it.

Reply:
Yes, but the test code does not use containerInterfaceParser.